ui/components/app/modals/edit-approval-permission/edit-approval-permission.component.js

```diff
@@ -80,11 +80,11 @@ export default class EditApprovalPermission extends PureComponent {
           <div className="edit-approval-permission__edit-section__description">
             {t('allowWithdrawAndSpend', [origin])}
           </div>
-          <div className="edit-approval-permission__edit-section__option">
-            <div
-              className="edit-approval-permission__edit-section__radio-button"
-              onClick={() => this.setState({ selectedOptionIsUnlimited: true })}
-            >
+          <button
+            className="edit-approval-permission__edit-section__option"
+            onClick={() => this.setState({ selectedOptionIsUnlimited: true })}
+          >
+            <div className="edit-approval-permission__edit-section__radio-button">
```

Comment:
I am not sure if it is good idea to wrap `div` inside `button`.
Ref: https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button